.github/workflows/integration-tests.yml

```diff
@@ -311,6 +315,91 @@ jobs:
           else
             echo "run_command=./smoke/${{ matrix.product.name }}_test.go" >> "$GITHUB_OUTPUT"
           fi
+      - name: Check for "enable tracing" label
+        id: check-label
+        run: |
+          label=$(jq -r '.pull_request.labels[].name' "$GITHUB_EVENT_PATH")
+          if [[ "$label" == "enable tracing" ]]; then
+            echo "Enable tracing label found."
+            echo "trace=true" >> $GITHUB_OUTPUT
+          else
+            echo "Enable tracing label not found."
+            echo "trace=false" >> $GITHUB_OUTPUT
+          fi
+          echo "$PWD"
+      - name: Setup Grafana and OpenTelemetry
+        id: docker-setup
+        if: steps.check-label.outputs.trace == 'true' && matrix.product.name == 'ocr2' && matrix.image.tag-suffix == '-plugins'
+        run: |
+          # Create network
+          docker network create --driver bridge tracing
+
+          # Start Grafana
+          cd ./.github/tracing
+          docker run -d --network=tracing --name=grafana -p 3000:3000 -v $PWD/grafana-datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml -e GF_AUTH_ANONYMOUS_ENABLED=true -e GF_AUTH_ANONYMOUS_ORG_ROLE=Admin -e GF_AUTH_DISABLE_LOGIN_FORM=true -e GF_FEATURE_TOGGLES_ENABLE=traceqlEditor grafana/grafana:9.4.3
+
+          # Start Tempo
+          docker run -d --network=tracing --name=tempo -v ./tempo.yaml:/etc/tempo.yaml -v $PWD/tempo-data:/tmp/tempo grafana/tempo:latest -config.file=/etc/tempo.yaml
+
+          # Start OpenTelemetry Collector
+          docker run -d --network=tracing --name=otel-collector -v $PWD/otel-collector.yaml:/etc/otel-collector.yaml -p 4317:4317 otel/opentelemetry-collector:0.61.0 --config=/etc/otel-collector.yaml
+
+      - name: Generate port
+        id: generate-port
+        env:
+          GITHUB_PR_NUMBER: ${{ github.event.number }}
+        run: |
+          PORT_BASE=3001
+          MAX_PORT=8000 
+  
+          # Use PR number as offset. Given GitHub PRs are incremental, this guarantees uniqueness for at least 5000 PRs.
+          OFFSET=$GITHUB_PR_NUMBER
+          echo "PR Number: $OFFSET"
```

Comment:
mv to after the mod

Reply:
yeah, i was cryptic. i'm asking for a log line that includes the value of the port (after modding offset)

now i see that it is actually there, but the echo's are going to different places. why is that - this is echo'd and the other is pipeline to a file

Reply:
I could remove this echo, but since workflows frequently tend to go run I though a bit of "debug" logging left behind might be helpful. 

The value of the port is echo'd to `GITHUB_OUTPUT`, which is how you can share state between steps in github workflows. This value will be output to the workflow terminal in any steps that consume it so the port and the PR number will both be visible.